dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImplTest.java

```diff
@@ -603,13 +604,19 @@ public void testSearch () throws Exception {
 
         //*****************************************************************
         //Build a site search result in order to add it to the index
-        ContentletVersionInfo versionInfo = APILocator.getVersionableAPI().getContentletVersionInfo(testHtmlPage.getIdentifier(), testHtmlPage.getLanguageId());
+        Optional<ContentletVersionInfo> versionInfo = APILocator.getVersionableAPI().getContentletVersionInfo(testHtmlPage.getIdentifier(), testHtmlPage.getLanguageId());
+
+        if(!versionInfo.isPresent()) {
```

Comment:
Just curious, is it OK to throw an exception here? It's because I've seen some places the expression `assertTrue(opt.isPresent())` is used in tests.

Reply:
I will do that instead. Good call